ddtrace/settings/debugger.py

```diff
@@ -0,0 +1,80 @@
+from envier import En
+
+from ddtrace import config
+from ddtrace.internal.agent import get_trace_url
+from ddtrace.internal.constants import DEFAULT_SERVICE_NAME
+from ddtrace.internal.utils.config import get_application_name
+from ddtrace.version import get_version
+
+
+DEFAULT_MAX_PROBES = 100
+DEFAULT_GLOBAL_RATE_LIMIT = 100.0
+
+
+def derive_tags(c):
+    # type: (En) -> str
+    _tags = dict(env=config.env, version=config.version, debugger_version=get_version())
+    _tags.update(config.tags)
+
+    return ",".join([":".join((k, v)) for (k, v) in _tags.items() if v is not None])
+
+
+class DebuggerConfig(En):
+    __prefix__ = "dd.debugger"
+
+    service_name = En.d(str, lambda _: config.service or get_application_name() or DEFAULT_SERVICE_NAME)
+    probe_url = En.d(str, lambda _: get_trace_url())
+    snapshot_intake_url = En.d(str, lambda _: get_trace_url())
```

Comment:
Nit: if this is concatenated with `_snapshot_intake_endpoint`, then is should be named `snapshot_intake_host`?

Not important since it's not configured externally. Maybe also prefix with `_`?

Reply:
I think this is in line with what other products do in the library (e.g. the tracer), so I'll just add the `_` if that's ok?